### ISSUE TITLE
Unity 5.4b16 Native Motion Vector usage

### ISSUE THREAD

**Issue by ArieLeo** [playdeadgames/temporal#5]
Is this effect already using unity native motion vector or still using custom one?

**Comment by Xerxes1138** [comment]
From what I've seen it use a custom one. 

Works great so far for me.

**Comment by pointcache** [comment]
Guys please make it work with native unity vectors!

**Comment by Invertex** [comment]
Agreed, it would be great to have it adjusted to work with Unity's built-in vectors now, which are automatically enabled for objects, and I'm sure they probably are generated a lot more efficiently since it's internal functionality.

But also,  PlayDead aren't selling this so they don't have much obligation, so maybe someone else will get around to it!

The TAA effect that's currently in a branch of the CinematicEffects bitbucket is decent, but still not to the level of quality this asset is, more noticeable jitter and doesn't handle further away edges as well. But I'm sure it will improve with time guys!

**Comment by Xerxes1138** [comment]
I've jsut added Motion Vector support for one of my effect so far it works great.
So I think I can make it work for this too.

EDIT : One problem though is that the motion vector is affected by the Frustum Jitter.
